src/__tests__/flow-spec/IBlock.spec.ts

```diff
@@ -188,49 +187,19 @@ describe('IBlock', () => {
         type: 'test',
         name: 'test',
         exits: [],
+        ui_metadata: {canvas_coordinates: {x: 10, y: 10}},
         config: {
           set_contact_property: {
             property_key: 'foo',
             property_value: 'bar',
-          } as ISetContactPropertyBlockConfig,
-        },
+          },
+        } as ISetContactPropertyBlockConfig,
       } as IBlock
       setContactProperty(block, context)
       const property = context.contact.getProperty('foo')
       expect(typeof property).toBe('object')
       expect((property as IContactProperty).__value__).toBe('bar')
     })
-
-    it('should set an array of contact properties', () => {
-      dataset = createDefaultDataset()
-      const context = Object.assign({}, cloneDeep(dataset.contexts[1]))
-      context.contact = new Contact()
-      const block = {
-        uuid: 'block-123',
-        type: 'test',
-        name: 'test',
-        exits: [],
-        config: {
-          set_contact_property: [
-            {
-              property_key: 'foo',
-              property_value: 'bar',
-            } as SetContactProperty,
-            {
-              property_key: 'baz',
-              property_value: 'qux',
-            } as SetContactProperty,
-          ],
-        },
-      } as IBlock<ISetContactPropertyBlockConfig>
-      setContactProperty(block, context)
-      const property1 = context.contact.getProperty('foo')
-      expect(typeof property1).toBe('object')
-      expect((property1 as IContactProperty).__value__).toBe('bar')
-      const property2 = context.contact.getProperty('baz')
-      expect(typeof property2).toBe('object')
-      expect((property2 as IContactProperty).__value__).toBe('qux')
-    })
```

Comment:
If we can't do this, will it cause any problems? How about for the AdvancedSelectOneBlock from our internal spec/custom blocks and runners, which selects a row from a CSV, and then can set each cell from that row to multiple mapped ContactProperties?

Reply:
the spec here should reflect the community version, that why Mark removed it https://github.com/FLOIP/flow-runner/commit/410546baff491a15e75134f307e65ae30d2f259d.

For our internal custom blocks, I think we have a plan to customize Interfaces/JsonSchema, but not in flow-runner repo. 
@jcbashdown: Can you remind us how this customization will be? thanks

Reply:
Hi @safydy-r for the customisation we will do to validate expressions, I don't yet have an approach in mind. This is what I said in the spike ticket:

>Ensure that we can continue to use the flow-runner schema and this schema is also still usable in places that don't know about the user defined ajv "expression" format

>  - If this is not possible then propose a solution for how these can safely diverge in specific contexts.

So part of that spike will be investigating how to do this. I don't have a suggestion yet as I think we need to get the ajv bit working first in order to understand how the schema and ajv work together.